root/defaults/proxy.conf

```diff
@@ -14,10 +14,13 @@ proxy_connect_timeout 240;
 proxy_set_header Host $host:$server_port;
 proxy_set_header X-Real-IP $remote_addr;
 proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-proxy_set_header X-Forwarded-Proto https;
+proxy_set_header X-Forwarded-Proto $scheme;
```

Comment:
This one is not necessary. We actually want to send https when we use ssl anyway

Reply:
That makes sense, this was something I did in other nginx environments to accommodate http when no SSL was being used, but I guess that doesn't really apply here in context.